meta/data.go

```diff
@@ -433,6 +438,10 @@ func (data *Data) DeleteShardGroup(database, policy string, id uint64) error {
 	for i := range rpi.ShardGroups {
 		if rpi.ShardGroups[i].ID == id {
 			rpi.ShardGroups[i].DeletedAt = time.Now().UTC()
+			for _, shard := range rpi.ShardGroups[i].Shards {
```

Comment:
Again, I think it makes more sense to log from where `DeleteShardGroup` is called.